apps/vms_term_sock.c

```diff
@@ -434,12 +434,12 @@ static int CreateSocketPair (int SocketFamily,
     /*
     ** Now issue the connect
     */
-    memset ((char *) &sin, 0, sizeof (sin)) ;
+    memset ((char *) &sin, 0, sizeof(sin)) ;
```

Comment:
While you're at it, why not remove space prior `&sin`?

Reply:
There's more space cleanup to do in that particular file.  Might almost be worth a PR of its own...

Reply:
Right, that's what I already suggested in #4875 :-)